### PR TITLE
add group and group member

### PR DESCRIPTION
**Pull request by fizikurniawan**

### CODE REVIEW

File: api/app/module/group/group.controller.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    return res.sendSuccess()
+  }
+
+  async addMember(req, res) {
+    const { user_uuid, group_uuid } = req.body
+
+    const requires = ['user_uuid', 'group_uuid']
+    let errors = validator.checkBody(req.body, requires, req.strings)
+    if (errors) return res.sendError(errors)
+
```
Comment (born2stuck) [inline api/app/module/group/group.controller.js:109]: validate only admin/group owner can perform this action

Comment (fizikurniawan) [reply api/app/module/group/group.controller.js:109]: OK ASAP

File: api/app/model/group.model.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+class GroupModel {
+  constructor() {
+    this.dbConnection = new DBConnection()
+    this.query = new Query()
+    this.table = 'groups'
+  }
+
+  async getGroupByUser(userId) {
+    const query = `SELECT * FROM ${this.table} WHERE user_uuid = $1`
+    const result = await this.dbConnection(query, [userId])
```
Comment (born2stuck) [inline api/app/model/group.model.js]: missing `.query()`

Comment (fizikurniawan) [reply api/app/model/group.model.js]: Latest commit

**Comment by born2stuck** [comment]
LGTM!

**Review by born2stuck — APPROVED** [review]

**Pull request merged**